saleor/extensions/plugins/avatax/plugin.py

```diff
@@ -292,17 +350,73 @@ def assign_tax_code_to_object_meta(
     def get_tax_code_from_object_meta(
         self, obj: Union["Product", "ProductType"], previous_value: Any
     ) -> TaxType:
-        if not self._enabled:
+        self._initialize_plugin_configuration()
+
+        if not self.active:
             return previous_value
         tax = obj.get_meta(namespace=META_NAMESPACE, client=META_FIELD)
         return TaxType(code=tax.get("code", ""), description=tax.get("description", ""))
 
     def show_taxes_on_storefront(self, previous_value: bool) -> bool:
-        if not self._enabled:
+        self._initialize_plugin_configuration()
+
+        if not self.active:
             return previous_value
         return False
 
     def taxes_are_enabled(self, previous_value: bool) -> bool:
-        if not self._enabled:
+        self._initialize_plugin_configuration()
+
+        if not self.active:
             return previous_value
         return True
+
+    @classmethod
+    def _get_default_configuration(cls):
+        defaults = {
+            "name": cls.PLUGIN_NAME,
+            "description": "",
+            "active": False,
+            "configuration": [
+                {
+                    "name": "Username or account",
+                    "value": "",
+                    "type": ConfigurationTypeField.STRING,
+                    "help_text": "Provide user or account details",
```

Comment:
This should be also marked for translation I guess. Btw how are we going to switch the language when returning content like this in the API? The dashboard would be available in different languages. Should [we manually activate a language ](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.translation.activate) in the resolver? cc @patrys

Reply:
If we want the client to pass the language to the resolver, we can temporarily activate a language using `with translation.override(...):`.